core/dbt/parser/sources.py

```diff
@@ -269,11 +269,6 @@ def parse_source_test(
             tags=tags,
             column_name=column_name
         )
-        # we can't go through result.add_node - no file... instead!
-        if node.config.enabled:
```

Comment:
Same here. What does this have to do with the rename? Honestly asking to learn

Reply:
This is the fix for #3347, in which we noticed a while ago that source tests are added to the manifest _twice_.

Things I don't know:
- why that doesn't cause a functional error currently
- why that _started_ causing a functional error when I made the other changes in this PR, i.e. renamed things

Despite not knowing those things, I feel reasonably good about making this fix. I can move the commit (0adec32) into a separate PR if that would make things clearer.